src/ApiCompat/Microsoft.DotNet.ApiCompat.Task/MSBuildCompatibilityLogger.cs

```diff
@@ -40,13 +38,11 @@ private bool LogSuppressableMessage(MessageLevel messageLevel, Suppression suppr
             if (_suppressionEngine.IsErrorSuppressed(suppression))
                 return false;
 
-            _suppressionWasLogged = true;
-
             _log.Log(new Message(messageLevel, string.Format(format, args), code));
 
             return true;
         }
 
-        public bool SuppressionWasLogged => _suppressionWasLogged;
+        public bool SuppressionWasLogged => _log.HasLoggedErrors;
```

Comment:
Sorry for the confusion. Can you please revert this change to what you had before? Asking as with the proposed logging consolidation with GenAPI we will be able to log errors that aren't related to suppressions.